src/lockstep/models/actionresultmodel.py

```diff
@@ -0,0 +1,30 @@
+#
+# Lockstep Platform SDK for Python
+#
+# (c) 2021-2022 Lockstep, Inc.
+#
+# For the full copyright and license information, please view the LICENSE
+# file that was distributed with this source code.
+#
+# @author     Lockstep Network <support@lockstep.io>
+# @copyright  2021-2022 Lockstep, Inc.
+# @link       https://github.com/Lockstep-Network/lockstep-sdk-python
+#
+
+
+from dataclasses import dataclass
+
+@dataclass
+class ActionResultModel:
+    """
+    Represents the result of an action.
+
+    In the Lockstep API, an Action is returned when an API call does not
+    produce any data but does produce messages that can be useful in
+    understanding what work was performed. You may use the messages text
+    to display user visible error messages or the results of various
+    operations.
+    """
+
+    messages: list[str] | None = None
```

Comment:
I noticed the our setup.cfg indicating support for >= 3.6 but the use of the union pipe operator just got added in 3.10. I can confirm that trying to run this locally causes problems with 3.6. I will follow up with suggestions on the main generator. In the current state we would need to change the min version requirement to 3.10